lib/dev3/slack_messenger/http_client.ex

```diff
@@ -32,7 +32,8 @@ defmodule Dev3.SlackMessenger.HTTPClient do
                           review_submitted:        ReviewSubmitted,
                           tagged_in_issue:         TaggedInIssue,
                           tagged_in_issue_comment: TaggedInIssueComment,
-                          rate_limit_exceeded:     RateLimitExceededResponse}
+                          rate_limit_exceeded:     RateLimitExceededResponse,
+                          listrepos_response:      ListreposResponse}
```

Comment:
To keep consistency with other slash command messages : `list_repos_response` and `ListReposResponse`